src/components/data/EntitiesContent.js

```diff
@@ -481,6 +488,16 @@ const EntitiesContent = ({
           ]),
         deleteColumnUpdateMetadata
       }),
+      addingEntity && h(AddEntityModal, {
+        entityType: entityKey,
+        attributeNames: entityMetadata[entityKey].attributeNames,
+        entityTypes: _.keys(entityMetadata),
+        workspaceId: { namespace, name },
+        onDismiss: () => setAddingEntity(false),
+        onSuccess: () => {
+          setRefreshKey(_.add(1))
+        }
```

Comment:
Nit: We could save some vertical space:

```suggestion
        onSuccess: () => setRefreshKey(_.add(1))
```